ark/utils/data_utils.py

```diff
@@ -85,13 +89,21 @@ def label_cells_by_cluster(fovs, all_data, label_maps, fov_col=settings.FOV_ID,
         relabeled_img_array = relabel_segmentation(labeled_img_array, labels_dict)
         img_data.append(relabeled_img_array)
 
-    return xr.DataArray(img_data, coords=[fovs, range(img_data[0].shape[0]),
-                                          range(img_data[0].shape[1])],
-                        dims=["fovs", "rows", "cols"])
+    return xr.DataArray(
+        img_data,
+        coords=[fovs, range(img_data[0].shape[0]), range(img_data[0].shape[1])],
+        dims=["fovs", "rows", "cols"],
+    )
 
 
-def generate_cell_cluster_mask(fovs, base_dir, seg_dir, cell_consensus_name,
-                               cell_cluster_col='cell_meta_cluster', seg_suffix='_feature_0.tif'):
+def generate_cell_cluster_mask(
+    fovs,
+    base_dir,
+    seg_dir,
```

Comment:
However, ones like this, that just take a list of items and puts them all on their own line, isn't actually that helpful. Willing to hear other opinions though